### ISSUE TITLE
Accessibility - Unable to tab through top level TOC entries

### ISSUE THREAD

**Issue by drussi01** [dotnet/docfx#8123]
**Operating System**: (`Windows` or `Linux` or `MacOS`)
Windows

**DocFX Version Used**: 

**Template used**: (`default` or `statictoc` or contain custom template)
default

**Steps to Reproduce**:

1. Generate docfx site (can repro on the official docfx site https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html)
2. Click an entry in the TOC on the left like Getting Started
3. Use tab to try to navigate to the Content entry in the TOC

**Expected Behavior**:
For accessibility you should be able to tab around to all items on the website

**Actual Behavior**:
You can only tab between the sub items in the list and not from Getting Started to the Content Entry

**Comment by paulushub** [comment]
@drussi01 Thanks for the head up. I followed this, tested the `docs.microsoft.com` site and this accessibility feature is not supported as well. It is, however, supported on `developer.android.com`.

Tested links: 
https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlreader?view=net-6.0
https://developer.android.com/reference/com/android/installreferrer/packages
